app/apps/addresses/models.py

```diff
@@ -64,34 +69,40 @@ def __str__(self):
             )
         return self.bag_id
 
-    def get(bag_id):
+    def get_or_create(bag_id):
         return Address.objects.get_or_create(bag_id=bag_id)[0]
 
-    def save(self, *args, **kwargs):
+    def get_bag_address_data(self):
+        # When moving the import to the beginning of the file, a Django error follows:
+        # ImproperlyConfigured: AUTH_USER_MODEL refers to model 'users.User' that has not been installed.
         from utils.exceptions import DistrictNotFoundError
 
         try:
-            bag_data = do_bag_search_id(self.bag_id)
-            result = bag_data.get("results", [])
+            bag_search_response = do_bag_search_id(self.bag_id)
+            bag_search_results = bag_search_response.get("results", [])
         except Exception:
-            result = []
```

Comment:
Je hebt hier een generieke Exceptie catch, maar je doet er eigenlijk niets mee.
De default waarde van bag_search_results kun je ook voor de try zetten.

Reply:
Met een `pass` onder `except Exception` kunnen excepties onopgemerkt blijven. Is dat by design wat jou betreft? Anders is het beter om de try/except helemaal weg te halen. Als er dan nog iets stuk gaat krijg je in ieder geval een error. Maar het kan ook zijn dat je die wilt onderdrukken dus.